packages/better-php-doc-parser/src/PhpDocNodeFactory/AbstractPhpDocNodeFactory.php

```diff
@@ -49,6 +49,7 @@ protected function resolveContentFromTokenIterator(TokenIterator $tokenIterator)
 
     protected function resolveFqnTargetEntity(string $targetEntity, Node $node): string
     {
+        $targetEntity = $this->getCleanedUpTargetEntity($targetEntity);
```

Comment:
I moved it to this place, it might be better to clean it up in the very beginning